app/models/user.rb

```diff
@@ -7,7 +7,7 @@ class User < ApplicationRecord
   validates :name, presence: true
 
   def total_fines
-    total = checkouts.inject(0) { |total, checkout| total + checkout.fine }
+    total = checkouts.inject(0) { |total, checkout| total + checkout.calculate_fine }
```

Comment:
I think you can use `sum` for this. Note that I'm talking about Ruby's sum and not Rails. The Rails one is for ActiveRecord relations